tests/mocks.go

```diff
@@ -58,8 +58,16 @@ func (mfs *MockFileService) HandleTaskPackage(taskId, userId, submissionNumber i
 		return services.TaskDirConfig{}, fmt.Errorf("failed to copy task files: %w", err)
 	}
 
-	userSolution := filepath.Join(mockUserSolutionDir, strconv.FormatInt(submissionNumber, 10)+".cpp")
-	destSolution := filepath.Join(dirPath, "solution.cpp")
+	var userSolution string
+	var destSolution string
+
+	if submissionNumber < 5 {
+		userSolution = filepath.Join(mockUserSolutionDir, strconv.FormatInt(submissionNumber, 10)+".cpp")
+		destSolution = filepath.Join(dirPath, "solution.cpp")
+	} else {
+		userSolution = filepath.Join(mockUserSolutionDir, strconv.FormatInt(submissionNumber, 10)+".py")
+		destSolution = filepath.Join(dirPath, "solution.py")
+	}
```

Comment:
bruh, should not be like that. You should add some internal variable or switch to indicate whether you want to get cpp or py solution. Definetly not using count